src/LibraryManagementSystemApplication.java

```diff
@@ -42,6 +42,35 @@ static void addBook(String title, String author, String ISBN, String pageNumber)
 
 
 
+    static void updatePatronInfo(String fullName, String identityNumber, String email, String password){
+        System.out.println("Enter ID of patron to be updated: ");
```

Comment:
you don't need to get input from user since your using method parameters. this is wrong.